src/main/io/gps.c

```diff
@@ -1985,4 +1985,9 @@ float getGpsDataIntervalSeconds(void)
     return gpsDataIntervalSeconds;
 }
 
+baudRate_e getGPSPortActualBaudRateIndex(void)
```

Comment:
Isn't it enough tu return baudrate, not index?

Reply:
The baud rate index seems to be currency used in a lot of other places. For example, the function used to get the configured GPS baud rate returns the index and a lot of functions in the serial library and elsewhere deal in indexes. I thought it good to maintain consistency with that convention for that getter.

The code in cli.c at lines 4749-4753 prints the two baud rates, so I also thought that code block would look more consistent to deal in indexes and have the two statements perform the similar array lookup, rather than having them use two different approaches.